source/Lib/CommonLib/arm/InitARM.cpp

```diff
@@ -73,6 +73,8 @@ void InterpolationFilter::initInterpolationFilterARM()
   switch( vext )
   {
   case NEON:
+  case SVE:
+  case SVE2:
     _initInterpolationFilterARM<NEON>();
```

Comment:
A switch case won't work in this case, right? Because you want to call the `<NEON>` init always, and than potentially overwrite some function pointers in `<SVE>` or `<SVE2>` init functions. Maybe, considering the approach here, it would be better to have 3 if statements, e.g.:

```cpp
if( vext >= NEON ) _init...ARM<NEOM>();
if( vext >= SVE ) _init...ARM<SVE>();
if( vext >= SVE2 ) _init...ARM<SVE2>();
```

Reply:
Thanks for the suggestion, that works much more cleanly. Done.